### ISSUE TITLE
Desktop timezone issue

### ISSUE THREAD

**Issue by nicktoggl** [toggl/toggldesktop#2160]
A few reports or the desktop app not choosing the correct time or date in app but appearing correct in browser.

> Switzerland we add 1 hour. But on Windows 10, this is considered as UTC;1 not UTC;2. On Toggl.com it;s considered as UTC;2.
> 
> The result is that in the desktop app, I create an entry at 5PM (UTC), it;s marked 6PM when it;s actually 7PM but on Toggl.com it;s correctly set as 7PM.

/case/450995

**Comment by nicktoggl** [comment]
Screenshot included case/451352

"Here I found another bug. Pretty strange one. I cannot sent the date of my entry to the 5th of April. It stays on 4th of April. I had to delete and re-create the entry...

To reproduce that I simply create an entry click the Start/Stop button and then edit it and set the date to 05/04/2017. The entry is placed on 04/04/2017.

My date setting is DD.MM.YYYY but in the desktop app it&#39;s not taken in consideration."
